src/azure-firewall/azext_firewall/_validators.py

```diff
@@ -143,3 +143,15 @@ def validate_rule_group_collection(namespace):
     if namespace.target_fqdns is not None and namespace.fqdn_tags is not None:
         raise CLIError('usage error: --target-fqdns | --fqdn-tags')
     return namespace
+
+
+def validate_private_ranges(namespace):
+    namespace.private_ranges = ', '.join(namespace.private_ranges)
+
+
+def validate_threat_intel_whitelist_ip_addresses(namespace):
+    namespace.ip_addresses = ', '.join(namespace.ip_addresses)
```

Comment:
It that what validator targets to?

Reply:
yes. It's a strange server. it accepts string with `,` to represent a list instead of an array.....

Reply:
No, I mean validator should be used to validate user input values not re-format them.
Is validator designed for this?

Reply:
Well, good question. From a bigger view, making the arguments validate also sounds reasonable. I'm not sure the original design spec. At least, in network module, it's common to re-format them.